Build failure on ubuntu-trusty

As reported in https://trac.sagemath.org/ticket/27952#comment:70
and as can be seen in GH Actions.

Other than that, there seem to be lots of failures that are not under the control of Normaliz. E.g.:
- `cygwin (minimal)` fails building Flint 2.5.2
- `cygwin (standard)` fails building e_antic-0.1.7
- `docker (debian-buster, standard)` also fails building e_antic-0.1.7
- `docker (ubuntu-trusty, minimal)` fails in a GitHub action setting with the rather unhelpful title `Run set -o pipefail; EXTRA_DOCKER_BUILD_ARGS="--build-arg USE_MAKEFLAGS=\"-k V=0 SAGE_NUM_THREADS=3\"" tox -e $TOX_ENV -- $TARGETS 2>&1 | sed "/^configure: notice:/s|^|::warning file=artifacts/$LOGS_ARTIFACT_NAME/config.log::|;/^configure: warning:/s|^|::warning file=artifacts/$LOGS_ARTIFACT_NAME/config.log::|;/^configure: error:/s|^|::error file=artifacts/$LOGS_ARTIFACT_NAME/config.log::|;"` ... Looking into this, I just found "core dump" error while starting up Docker (?) After digging around more, I finally also found what looks like parts of a compiler error this one is affected by that `isnan` issue mentioned on the SageMath Trac. But fishing this out of the log is really cumbersome
- `docker (centos-7, minimal)` also seems to be related to `isnan`, so let's see whether PR #347 helps with it (in
- `docker (linuxmint-17, standard)` failed building Normaliz... somehow? The error log is too short to tell, at  least I couldn't find the actual error? The log for normaliz seems to start like this:
```
  [normaliz-git]   make[3]: *** [dynamic/dynamic.o] Error 1
  [normaliz-git]   make[3]: *** [maxsimplex/maxsimplex.o] Error 1
  [normaliz-git]   make[3]: *** [libnormaliz/collection.lo] Error 1
  ...
```
so it cuts off right after the actual error?

I gave up at that point, as I found it far too frustrating to dig through these logs :-(

> * `docker (linuxmint-17, standard)` failed building Normaliz... somehow? The error log is too short to tell, at  least I couldn't find the actual error? The log for normaliz seems to start like this:
> 
> ```
>   [normaliz-git]   make[3]: *** [dynamic/dynamic.o] Error 1
>   [normaliz-git]   make[3]: *** [maxsimplex/maxsimplex.o] Error 1
>   [normaliz-git]   make[3]: *** [libnormaliz/collection.lo] Error 1
>   ...
> ```
> 
> so it cuts off right after the actual error?

There are two ways how you can see more:

1) Open " Print out logs for immediate inspection". This will show all logs of failing packages

2) When all builds have finished, you will be able to download all logs as a big zip file.

--- a/source/libnormaliz/matrix.h
+++ b/source/libnormaliz/matrix.h
@@ -644,7 +644,7 @@ Matrix<Number> LLL_red(const Matrix<Number>& U, Matrix<Integer>& T, Matrix<Integ
             /* cout << "MMMMM " << i << " " << j << " " << M[i][j] << endl;
             cout << i << "---" << G[i];
             cout << j << "---" << G[j];*/
-            if (isnan(M[i][j])) {
+            if (std::isnan(M[i][j])) {
                 T = Tinv = Matrix<Integer>(U.nr_of_rows());
                 return U;
             }

